docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_adding_an_integration_test.adoc

```diff
@@ -1,6 +1,6 @@
 = Adding an integration test
 
-You can add an integration test to ensure that individual components correctly integrate to form a whole application. {ProductName} runs integration tests on the container images of components before it deploys those images.     
+You can add an integration test to ensure that individual components correctly integrate to form a whole application. {ProductName} runs integration tests on the container images of components before it deploys those images.
```

Comment:
```suggestion
in {ProductName}, you can add integration tests to verify that the individual components of your application integrate correctly, forming a complete and functional application. {ProductName} runs these integration tests on the container images of components before their deployment.
```

Reply:
actually I didn't change this, I only accidentally pushed whitespace removal done by IDE. But can include it, no problem